certified-connectors/PartnerLinq/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
```

Comment:
@FarhanJamilVSI the type of username should be string, so it will visible for users.